### PR TITLE
Human Organs++

### PR DESCRIPTION
**Pull request by ghost**
Refer to issue #1183 .
### Intent of your Pull Request

Start of adding organs for Adam's BIG REWORK.

Todo:
- [ ] Receive input.
- [ ] Liver toxin damage.
- [ ] Liver removal.
- [ ] Lung removal.
- [ ] Lung damage with smoke/fire/chemicals.
- [ ] Eye removal.
- [ ] Integrate current eye damage with organs.
- [x] Fail at coding.
#### Changelog

:cl:
rscadd: Livers, lungs, and eyes are now real human(oid) organs.
/:cl:

### CODE REVIEW

**Comment by Kmc2000** [comment]
Poke if sprites needed

File: code/modules/surgery/organs/organ_internal.dm [hunk 1]
```diff
@@ -193,6 +193,34 @@
...
 	return S
 
+/obj/item/organ/internal/liver
+	name="liver"
+	icon_state="liver"
+	zone="chest"
+	slot="liver"
+	var/dysfunct = 0
+
+/obj/item/organ/internal/appendix/update_icon()
```
Comment (Super3222) [inline code/modules/surgery/organs/organ_internal.dm:203]: Wait what? Appendix?

File: code/modules/surgery/organs/organ_internal.dm [hunk 2]
```diff
@@ -193,6 +193,34 @@
...
+	if(inflamed)
+		icon_state = "livershitter
```
Comment (Super3222) [inline code/modules/surgery/organs/organ_internal.dm:205]: This is causing the merge conflicts. You are missing a ".

**Comment by oisin100** [comment]
If your going to continue coding for us @Groxic you should recreate this PR on the new repository.

**Pull request closed without merging**